.github/workflows/doc-issues.yml

```diff
@@ -0,0 +1,67 @@
+name: Create doc issues on PR merge
+
+# Action that creates issues in cockroachdb/docs when a PR is merged from
+# cockroachdb/cockroach in the master or release-* branch and has at
+# least one valid release note in PR commits.
+
+on:
+  pull_request:
+    branches:
+      - master
+      - 'release-*'
+
+    types: [closed]
+
+jobs:
+  create-doc-issue:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event.pull_request.merged == true }}
+    env:
+      PR_TITLE: ${{ github.event.pull_request.title }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
+    steps:
+      - name: Get PR commits
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@55b867b9b28954e6f5c1a0fe2f729dc926c306d0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Iterate through all commit messages on this PR.
+      # For each commit message, check whether it contains at least one release note.
+      # Extract the type(s) of release note(s) and check whether there exists
+      # at least one type that is not "none," "bug"-, or "performance"-related.
+      # If so, format the release note(s) and create a docs issue.
+      - name: Parse PR commits 
+        id: commits
+        uses: stefanprodan/kube-tools@0d93a66a9a3e96d912671e1fd3b5a27b6df6e5ef
+        env:
+          COMMITS: ${{ steps.get-pr-commits.outputs.commits }}
+        with:
+          command: |
```

Comment:
Have you read https://securitylab.github.com/research/github-actions-untrusted-input/? I don't have a repro, but I don't have a great deal of confidence that this isn't vulnerable to any of the abuses described in that article. @yzdocs, can you provide a little clarity here? 

https://securitylab.github.com/research/github-actions-preventing-pwn-requests/ is also relevant, but I *think* we're okay from this perspective? I think if this action only runs when pull requests are closed, then GitHub's normal security measures (not passing secrets to actions triggered by pull requests) should probably be fine, although we should do our due diligence there too.